pipeline_dp/combiners.py

```diff
@@ -507,6 +559,17 @@ def create_compound_combiner(
         combiners.append(
             PrivacyIdCountCombiner(
                 CombinerParams(budget_privacy_id_count, aggregate_params)))
+    if pipeline_dp.Metrics.ARRAY_SUM in aggregate_params.metrics:
+        budget_vector_sum = budget_accountant.request_budget(
+            mechanism_type, weight=aggregate_params.budget_weight)
+        combiners.append(
+            VectorSumCombiner(
+                CombinerParams(budget_vector_sum, aggregate_params)))
+        if pipeline_dp.Metrics.SUM in aggregate_params.metrics or \
+            pipeline_dp.Metrics.MEAN in aggregate_params.metrics or \
+            pipeline_dp.Metrics.COUNT in aggregate_params.metrics or \
+            pipeline_dp.Metrics.VARIANCE in aggregate_params.metrics:
+            raise ValueError("Do not combine scalar with vector metrics")
```

Comment:
"Vector sum can not be computed together with scalar metrics, like sum, mean etc"